### ISSUE TITLE
Track per-gem project counts as well as pure install counts

### ISSUE THREAD

**Issue by lynncyrin** [rubygems/rubygems#3203]
> CocoaPods launched a feature where they collect not just download stats, but also project-usage stats: http://blog.cocoapods.org/Stats/
> 
> We could do this by sending a unique project identifier on each install command—probably the hash of the git URL of the git `upstream` remote (if it exists) or the `origin` remote otherwise.
> 
> That would let us know which gems are the most widely used, rather than just the most downloaded. Is there a difference? We have no way to find out right now.

**Comment by jeromedalbert** [comment]
There is basically no activity here for the past 8 years. Unless there are any objections, let's close this issue for now and re-opened if needed.

**Comment by deivid-rodriguez** [comment]
Yeah, I think https://github.com/rubygems/rubygems.org/issues/1335 is the best place to continue the discussion 👍.
